data/common.yaml

```diff
@@ -24,10 +25,14 @@ profile_system_auth::config::oddjobd_mkhomedir_conf: |
       </object>
     </service>
   </oddjobconfig>
-profile_system_auth::config::required_pkgs:
+profile_system_auth::config::removed_pkgs:
+  - "fprintd-pam"
   - "authconfig"
+profile_system_auth::config::required_pkgs:
+  - "authselect"
   - "oddjob"
   - "oddjob-mkhomedir"
+profile_system_auth::config::use_authconfig: false
```

Comment:
Minor, non-blocking suggestion: I think it makes sense that these lines be in data/os/RedHat.yaml (since the equivalent lines for EL7 are also under the data/os directory in hiera.